src/app/shared/post-capture-card/post-capture-card.component.ts

```diff
@@ -84,7 +87,8 @@ export class PostCaptureCardComponent implements OnInit {
     return this.asset$
       .pipe(
         first(),
-        concatMap(asset => this.diaBackendAssetRepository.remove$(asset))
+        concatMap(asset => this.diaBackendAssetRepository.remove$(asset)),
+        switchTapTo(this.diaBackendAssetRepository.refresh$())
```

Comment:
I don't think will refresh the PostCapture cards. Refresh the `DiaBackendTransactionRepository` instead.